src/store/index.js

```diff
@@ -1,10 +1,9 @@
 import { configureStore } from "@reduxjs/toolkit";
```

Comment:
Isn't better to change ` index.js` name to somethings else ex. `store.js`

Reply:
done

Reply:
done